prowler/providers/aws/services/elasticache/elasticache_redis_cluster_below_v6_auth_enabled/elasticache_redis_cluster_below_v6_auth_enabled.py

```diff
@@ -0,0 +1,27 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.elasticache.elasticache_client import (
+    elasticache_client,
+)
+
+
+class elasticache_redis_cluster_below_v6_auth_enabled(Check):
+    def execute(self):
+        findings = []
+        for cluster in elasticache_client.clusters.values():
+            if (
+                cluster.engine == "redis" and int(cluster.engine_version[0]) < 6
+            ):  # major.minor.patch
```

Comment:
What is the comment for?

Reply:
It was thought to clarify the string format of the version stored in `engine_version`. If it is unclear or unnecessary we can remove it.